influxdb/client.py

```diff
@@ -284,15 +285,18 @@ def write(self, data, params=None, expected_response_code=204,
         headers = self._headers
         headers['Content-type'] = 'application/octet-stream'
 
-        if params:
+        if params and 'precision' in params:
```

Comment:
Well, I thought I got exception from this but apparently missing key with get() gives Null.
You're right.